mod_locations.asciidoc

```diff
@@ -6,43 +6,44 @@
 
 *Data owner: `CPO`*
 
-The Location objects live in the CPO back-end system. They describe the charging locations of that operator.
+The Location objects live in the CPO back-end system. They describe the charging locations of one operator (or of multiple operators in case of virtual CPOs).
 
 *Module dependency:* the eMSP endpoint is dependent on the <<mod_tariffs.asciidoc#mod_tariffs_tariffs_module,Tariffs module>>
 
 [[mod_locations_flow_and_lifecycle]]
 === Flow and Lifecycle
 
-The Locations module has Locations as base object, Locations have EVSEs, EVSEs have Connectors.
+The Locations module has Location as base object. Locations can have multiple EVSEs and EVSEs can have multiple Connectors.
 With the methods in the <<mod_locations_emsp_interface,eMSP interface>>,
 Location information/statuses can be shared with the eMSP.
-Updates can be done to the Location, but also to only an EVSE or a Connector.
+Updates can be made to a whole Location, but also to only an EVSE or a Connector.
 
-When a CPO creates Location objects it pushes them to the eMSPs by calling <<mod_locations_put_method,PUT>> on the eMSPs Locations endpoint.
-Providers who do not support push mode need to call <<mod_locations_get_method,GET>> on the CPOs Locations endpoint to receive the new object.
+When a CPO creates Location objects, it pushes them to the eMSPs by calling <<mod_locations_put_method,PUT>> on the eMSPs Locations endpoint.
+Providers who do not support PUSH mode need to call <<mod_locations_get_method,GET>> on the CPOs Locations endpoint to receive the new object.
```

Comment:
Would all caps `PUSH` be more clear? Then I/we need to check the entire document for this. And then also for `PULL`.

Reply:
In my opinion it would be more clear as written in another comment above in more detail.